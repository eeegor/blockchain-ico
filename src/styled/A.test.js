import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { A } from '.';

describe('A', () => {
	test('renders a tag', () => {
		const tree = renderer.create(<A color="red" />).toJSON();
		expect(tree).toMatchSnapshot();
		expect(tree).toHaveStyleRule('color', 'red !important');
	});
});
