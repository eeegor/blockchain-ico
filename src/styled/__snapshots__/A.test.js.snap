// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A renders a tag 1`] = `
.c0 {
  font-size: 16px;
  -webkit-letter-spacing: 0.5px;
  -moz-letter-spacing: 0.5px;
  -ms-letter-spacing: 0.5px;
  letter-spacing: 0.5px;
  line-height: 26px;
  font-family: GTCinetype-Regular;
  color: #2B60FF;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 0.15s ease-out;
  transition: all 0.15s ease-out;
  color: red !important;
}

.c0:hover {
  color: #0031c4;
}

<a
  className="c0"
  color="red"
/>
`;
